.github/workflows/e2e-test.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
         uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: "12.x"
+          node-version: "16.x"
```

Comment:
`esbuild-plugin-postcss2` uses modern js syntax likes `args?.pluginData?.originalPath`, so we need the higher node to run it.